bf2fc6cc711aee1a0c2a/kas-fleet-manager
# added script and make target to process perf test resutls and output them in JSON schema

## Description
https://issues.redhat.com/browse/MGDSTRM-3479 - added script and make target to process perf test results and output them in JSON schema

## Description

## Verification Steps
1. Create a file with csv results in expected format in `test/performance/reports/perf_test_stats.csv`, e.g.:

```
Type,Name,Request Count,Failure Count,Median Response Time,Average Response Time,Min Response Time,Max Response Time,Average Content Size,Requests/s,Failures/s,50%,66%,75%,80%,90%,95%,98%,99%,99.9%,99.99%,100%
GET,/cloud_providers,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1
GET,/cloud_providers/aws/regions,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2
GET,/kafkas,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3
POST,/kafkas,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4,4
GET,/kafkas/[id],5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5,5
GET,/kafkas/[id]/metrics/query,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6
GET,/kafkas/[id]/metrics/query_range,7,7,7,7,7,7,7,7,7,7,7,7,7,7,7,7,7,7,7,7
GET,/kafkas?search,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8
GET,/openapi,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9
GET,/serviceaccounts,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10,10
,Aggregated,11,11,11,11,11,11,11,11,11,11,11,11,11,11,11,11,11,11,11,11
```

2. Run `make test/performance/process/csv `, confirm that there are no errors and that the csv was successfully converted to  `test/performance/reports/perf_test_stats.json` with data set as per csv

## Type of change


## Checklist (Definition of Done)
